examples/event-webhooks-local-flask/local_app/benchling_app/sync_data.py

```diff
@@ -0,0 +1,42 @@
+
+
+from benchling_api_client.webhooks.v0.beta.models.v2_entity_registered_event import (
+    V2EntityRegisteredEvent as V2EntityRegisteredEventBeta,
+)
+from benchling_sdk.apps.framework import App
+from benchling_sdk.apps.status.helpers import ref
+from benchling_sdk.models import (
+    AppSessionMessageCreate,
+    AppSessionMessageStyle,
+    AppSessionUpdateStatus,
+    CustomEntity,
+)
+
+from local_app.lib.postgresql import write_data
+
+
+def sync_event_data(app: App, entity_registered: V2EntityRegisteredEventBeta) -> None:
```

Comment:
@edwin-benchling This is the second file that's "new" and could use closer review.